apps/src/studio/spriteActions.js

```diff
@@ -0,0 +1,141 @@
+/** @file Actions that can be given to a playlab sprite to execute over a set time. */
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+ eqeqeq: true,
```

Comment:
...probably not?  They're jshint directives, the sort of thing we'd eventually like to remove from our files and apply more broadly anyway.  Also, this comment requires a special format since jshint reads it.